demisto_sdk/commands/postman_codegen/tests/postman_codegen_test.py

```diff
@@ -204,24 +205,92 @@ def test_duplicate_requests_check_duplicates_dont_exist(self):
         }
         duplicate_requests_check(names_dict)
 
+    test_files_path = os.path.join(git_path(), 'demisto_sdk', 'commands', 'postman_codegen', 'tests', 'test_files')
+    with open(os.path.join(test_files_path, 'shared_args_path.json')) as shared_args_path_file:
+        shared_args_path_items = json.load(shared_args_path_file)
+
+    more_then_one_paths = defaultdict(int, {'name': 2, 'description': 3, 'url': 3, 'method': 3})
+    one_for_each_paths = defaultdict(int, {'name': 1, 'id': 1, 'uid': 1})
+    complicated_chars_paths = defaultdict(int, {'name': 1, 'required': 5})
+    same_path_at_the_end = ({'collection.item.settings.name': 'item_settings_name',
+                             'collection.info.settings.name': 'info_settings_name',
+                             'collection.item.property.name': 'item_property_name'},
+                            defaultdict(int, {'name': 2}))
+    SHARED_ARGS_PATH = ((shared_args_path_items['more_then_one_items'], more_then_one_paths),
+                        (shared_args_path_items['no_path_items'], defaultdict(int)),
+                        (shared_args_path_items['one_for_each_items'], one_for_each_paths),
+                        (shared_args_path_items['complicated_chars_items'], complicated_chars_paths),
+                        same_path_at_the_end)
+
+    @pytest.mark.parametrize('flattened_json, shared_arg_to_split_position_dict', SHARED_ARGS_PATH)
+    def test_find_shared_args_path(self, flattened_json, shared_arg_to_split_position_dict):
+        """
+        Given
+        - Dictionary containing flattened json with raw body of a request
+
+        When
+        - There are arguments in the request which have the same name (suffix)
+
+        Then
+        - Returns arguments' dictionary with an entry for each argument name, that holds the minimum distinguishing shared path of
+        all arguments with the same name
+        """
+        for key, value in find_shared_args_path(flattened_json).items():
+            assert shared_arg_to_split_position_dict[key] == value
+
+    split_path1 = 'collection.item.settings.name'.split('.')
+    split_path2 = 'collection.info.settings.name'.split('.')
+    split_path3 = 'collection.item.property.name'.split('.')
+    split_path4 = 'set.item.property.name'.split('.')
```

Comment:
you can save them already as the string splitted to a list instead of using split

Reply:
Done